### ISSUE TITLE
Aya sign misplaced on WebKit browsers

### ISSUE THREAD

**Issue by forabi** [forabi/aQuran#5]
![image](https://cloud.githubusercontent.com/assets/1012761/2674386/a64c0a7a-c100-11e3-80e1-bb4bc6a5594d.png)
Ideally this should be fixed upstream.
Gecko has no issues rendering the Aya Sign character.

**Comment by ATouhou** [comment]
You could do something like

:before {
    content: '\06DD';
    position: absolute;
    top: 8px;
    left: -10px;
}

i.aya-number {
    position: relative;
}

i.aya-number:before {
    content: '\06DD';
    position: absolute;
    top: 8px;
    left: -10px;
}

to fix it... 

It's a great work you have made brother, sad to see you let this bug exist more than 1 year

**Comment by ATouhou** [comment]
the font should also be alittle bit smaller in .aya-number and bigger for the content element

**Comment by forabi** [comment]
The app is now deprecated. Please do not post issues/pull requests. I leave the source code available so that anyone can fork and fix. The reason I abandoned this app is a combination of not having enough time and the project becoming so bloated and requiring a major rewrite, probably getting rid of Angular in order to reduce the significant lag/frame drops.

**Comment by forabi** [comment]
Also, the fix for this is to add the aya number to the end of the Aya in JS. The WebKit bug is CSS-only.

**Comment by ATouhou** [comment]
May Allah make your life easy for you,  grant you time to benefit the ummah, and benefit yourself through this

**Comment by ahimta** [comment]
@forabi Thanks for all the effort you've put into this project so far.

I am not surprised you've stopped working on the project; I have a couple of abandoned projects myself.

What I've learned through the years is that working on any non-trivial software project and making it work well is never easy.

That is exactly why, in many cases, you shouldn't work on big projects for many reasons:
1. They require lots of time, so you'll have to either spend an unreasonably long time working on them or cut some corners and sacrifice quality to finish them, which in many cases ends up making the code and architecture just plain horrible.
2. Most big complex projects are harder to use and learn, so even after finishing them there is a big risk that they won't be good enough or just too much.
3. There's a huge gap between software that works and software that works well. Once you realize that you'll try to focus your efforts on smaller projects with better quality.
4. Practice makes perfect, however with big projects you'll practice (write code, design) too much that you'll become less efficient and accrue technical dept, pick up bad habits and hate Software Development at the same time.

**Comment by ATouhou** [comment]
@Ahimta  Agreed. On bigger projects and working alone, I always lose most motivation after completing about 50-70% of it.   

But structure is crucial, so developing on it will always be pleasent and "pretty". 

I hope we get the chance to work on something together in the future in Sha Allah. I wish to some day come and stay in Saudi, while working there :-)  I'm a fullstack developer (Primarily working with PHP, MySQL, HTML, CSS & JS)   located in Denmark.

**Comment by Ak-0** [comment]
I want to add that this was an excellent app and sad to see it go but that
seems like the direction FFOS is going. You guys seem to be really good
developers. I know website design is similar in terms of front-end and
back-end but developing an app is much more time consuming. I am willing to
help if you plan to develop a website.

**Comment by forabi** [comment]
Thank you all for your kind words and support.
I want to clarify some points:
- I am not a professional web developer. I study dentistry, and web development is just a hobby for me.
- I am graduating this year and had so little time to continue learning JS, but I did follow every piece of news I found interesting about ES6 and web app performance optimization.
- Looking back at the repo, I now realize I made some bad decisions, including the choice of CoffeeScript and Angular. But I have learned SO much working on this project that I feel more confident than ever about what makes a good web app. If I have the time to rewrite this app one day, I will probably choose vanilla JS without any framework, but making use of new ES6 features and Service Worker instead of AppCache. In fact I did set up a new repo (aQuran-Next) but I could not get to continue working on it.
